Tasks/Bartosh/Task2/Task2.py

```diff
@@ -24,12 +24,25 @@
     t = str(timestamp.hour)+':'+str(timestamp.minute)
 # 2 (or any key) for input from keyboard
 else:
-    while True:#validation that input data is in time range 
-      t = input ('press time in hh:mm format:')
-      if int(t.split(':')[0])<0 or int(t.split(':')[0])>23 or int(t.split(':')[1])<0 or int(t.split(':')[1])>59:
-        print("You should enter TIME in HH:MM F O R M A T:") #talking louder about the format
-      else:
+  t = input ('press time in hh:mm format:')
+  while True:
+    #find :
+    t_t = ':' in t
+    #find length
+    t_len = len(t) 
```

Comment:
I would suggest to check these things directly in the if operator and not spend additional variables to store it